src/v2/view-builder/components/EmailHeader.js

```diff
@@ -0,0 +1,12 @@
+import { BaseHeader } from '../internals';
+import { getIconClassNameForBeacon } from '../utils/AuthenticatorUtil';
+import { AUTHENTICATOR_KEY } from '../../ion/RemediationConstants';
+import HeaderBeacon from './HeaderBeacon';
+
+export default BaseHeader.extend({
+  HeaderBeacon: HeaderBeacon.extend({
+    getBeaconClassName() {
+      return getIconClassNameForBeacon(AUTHENTICATOR_KEY.EMAIL);
```

Comment:
Was wondering if we could just pass in authenticatorKey in `options` and combine `EmailHeader` and `OktaVerifyHeader` into `AuthenticatorHeader`. And just reuse it at different places by passing in a value in options.

Reply:
Good idea!! I'll look into it.